app/frontend/src/components/objectList/ShareObject.vue

```diff
@@ -61,12 +79,17 @@
               :value="showcasePageRoute"
             ></v-text-field>
 
-            <h2 class="mb-3">QR Code</h2>
+            <BaseCopyToClipboard
+              :copyText="showcasePageRoute"
+              snackBarText="Copied to clipboard"
+              tooltipText="Copy Share Link to clipboard" />
+
+            <!-- <h2 class="mb-3">QR Code</h2>
             <qrcode-vue
               :value="showcasePageRoute"
               size="250"
               level="H"
-            ></qrcode-vue>
+            ></qrcode-vue> -->
```

Comment:
Why are we removing the qrcode functionality? It should still be functional.

Reply:
the qr code is shown when you choose the option to share a public link.
i removed it from the 'share with a specific user' option because i found the UI a bit confusing.
I'll put it back in and we can get some more feedback on that.